libtwolame/encode.c

```diff
@@ -479,13 +479,8 @@ void write_bit_alloc(twolame_options * glopts, unsigned int bit_alloc[2][SBLIMIT
     int sb, ch;
 
     for (sb = 0; sb < sblimit; sb++) {
```

Comment:
I think that selection is clear enough for anyone knowing a little bit of layerII encoding.
Further changes, if any, should only be related to unrolling 'for' loops and increase speed, taken into account this function is called for each and every frame created.